recipes/r-ichorcna/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version="0.1.0.20180710" %}
+{% set revision="8021afa" %}
+{% set name="ichorCNA" %}
+{% set sha256sum="1155c0e4cd415489e3ea9b00f7fdfee9bf13f3f2aeec04ab0a32b137b8ec850f" %}
+
+package:
+  name: 'r-{{ name|lower }}'
+  version: '{{ version }}'
+source:
+  url: https://github.com/broadinstitute/ichorCNA/archive/{{ revision }}.tar.gz
+  sha256:  {{ sha256sum }}
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  host:
+    - r-base
+    - libgfortran-ng
```

Comment:
This does not belong into the host section - run should be enough.
Also please note that this is a bug, which we are working on in the r-base-feedstock.